packages/theme-toolkit/src/component-stories-utrecht.tsx

```diff
@@ -1907,15 +1903,241 @@ export const UTRECHT_COMPONENT_STORIES: ComponentStory[] = [
     ),
     detectTokens: {
       anyOf: [
-        'utrecht.table.caption.font-family',
         'utrecht.table.caption.font-size',
+        'utrecht.table.caption.font-weight',
+        'utrecht.table.caption.font-family',
         'utrecht.table.caption.color',
         'utrecht.table.caption.line-height',
         'utrecht.table.caption.text-align',
         'utrecht.table.caption.margin-block-end',
       ],
     },
   },
+  {
+    storyId: 'react-utrecht-table--header',
+    component: 'utrecht-table',
+    name: 'Utrecht Table with header',
+    render: () => (
+      <Table>
+        <TableCaption>Caption of the Table</TableCaption>
+        <TableHeader>
+          <TableRow>
+            <TableHeaderCell scope="col">Column Heading Cell 1</TableHeaderCell>
+            <TableHeaderCell scope="col">Column Heading Cell 2</TableHeaderCell>
+            <TableHeaderCell scope="col">Column Heading Cell 3</TableHeaderCell>
+          </TableRow>
+        </TableHeader>
+        <TableBody>
+          <TableRow>
+            <TableHeaderCell scope="row">Row Heading Cell 1</TableHeaderCell>
+            <TableCell>Data Cell 1</TableCell>
+            <TableCell>Data Cell 2</TableCell>
+          </TableRow>
+          <TableRow>
+            <TableHeaderCell scope="row">Row Heading Cell 2</TableHeaderCell>
+            <TableCell>Data Cell 3</TableCell>
+            <TableCell>Data Cell 4</TableCell>
+          </TableRow>
+          <TableRow>
+            <TableHeaderCell scope="row">Row Heading Cell 3</TableHeaderCell>
+            <TableCell>Data Cell 5</TableCell>
+            <TableCell>Data Cell 6</TableCell>
+          </TableRow>
+        </TableBody>
+      </Table>
+    ),
+    detectTokens: {
+      anyOf: [
+        'utrecht.table.header.background-color',
+        'utrecht.table.header.font-weight',
+        'utrecht.table.header.text-transform',
+        'utrecht.table.header.color',
+        'utrecht.table.header.border-block-end-color',
+        'utrecht.table.header.border-block-end-width',
+        'utrecht.table.header.sticky.background-color',
+        'utrecht.table.header.sticky.color',
+        'utrecht.table.header.sticky.border-block-end-color',
+        'utrecht.table.header.sticky.border-block-end-width',
```

Comment:
```suggestion
        'utrecht.table.header.background-color',
        'utrecht.table.header.font-weight',
        'utrecht.table.header.color',
        'utrecht.table.header.border-block-end-color',
        'utrecht.table.header.border-block-end-width',
```

`sticky.*` tokens verdienen een eigen story, anders kun je ze niet in actie zien
`text-transform` is discouraged, dus laten we die niet in de spotlight zetten.